bpf/process/types/basic.h

```diff
@@ -172,6 +185,11 @@ struct event_config {
 	 */
 	__u32 policy_id;
 	__u32 flags;
+	struct config_btf_arg btf_arg0[MAX_BTF_ARG_DEPTH];
+	struct config_btf_arg btf_arg1[MAX_BTF_ARG_DEPTH];
+	struct config_btf_arg btf_arg2[MAX_BTF_ARG_DEPTH];
+	struct config_btf_arg btf_arg3[MAX_BTF_ARG_DEPTH];
+	struct config_btf_arg btf_arg4[MAX_BTF_ARG_DEPTH];
```

Comment:
why not follow the user space part and have just `btf_arg[5][MAX_BTF_ARG_DEPTH]`
I did not see `btf_arg[1-4]` being touched elsewhere..